src/lifecycle.js

```diff
@@ -37,12 +38,12 @@ function ensureLifecycleFlag (target, component, name) {
 }
 
 /**
- * Parses an event definition and returns information about it.
- *
- * @param {String} e The event to parse.
- *
- * @returns {Object]}
- */
+* Parses an event definition and returns information about it.
+*
+* @param {String} e The event to parse.
+*
+* @returns {Object]}
+*/
```

Comment:
That was probably because Atom fucked up the formatting when pasting the doc block.